server/server.js

```diff
@@ -41,6 +43,23 @@ if (process.env.ENV === "PRODUCTION") {
   });
 }
 
+app.get("/authenticate/user-posts", (req, res) => {
```

Comment:
It might be better to move this to `api.js` and have the endpoint be `/api/user-posts`. (And because it's in `api.js`, the prefix `/api` will automatically be added!) Right now, it's fine as is as well, but the name is a lil mismatched because we're not really authenticating anything.

Reply:
Do you think it would be better if we move this code into a `routes/` folder and name it `auth.js` or something? I currently don't use `api.js` at all and created a `routes/` folder with `routes/post.js` and `routes/user.js` for the apis, I think it would be better for consistency :D

Reply:
Yeah now that I saw your PR, i think thats a good idea!